pkg/mgrconfig/config.go

```diff
@@ -166,6 +166,8 @@ type Config struct {
 	EnabledSyscalls []string `json:"enable_syscalls,omitempty"`
 	// List of system calls that should be treated as disabled (optional).
 	DisabledSyscalls []string `json:"disable_syscalls,omitempty"`
+	// List of syscalls that should not be mutated by the fuzzer (optional).
+	NoMutationSyscalls []string `json:"do_not_mutate_syscalls,omitempty"`
```

Comment:
It would be less confusing if the field name was similar to the json name (like in all other cases).

Also re. name -- maybe "immutable_calls"?

Reply:
Probably "immutable_syscalls", since above uses *_syscalls for names.

Another alternative: no_mutate_syscalls.

Pick whichever seems most intuitive.

Reply:
I will go for `no_mutate_syscalls` - I'm wary of the distinction between "mutable" and "mut**at**able", so I think this option is clearest.